GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -79,6 +79,13 @@ public override IEnumerable<EventListener> CreateTelemetryListeners(string provi
         {
             string gitBinRoot = this.GitInstallation.GetInstalledGitBinPath();
 
+            // If we do not have a git binary, then we cannot check if we should set up telemetry
+            // We also cannot log this, as we are setting up tracer.
+            if (string.IsNullOrEmpty(gitBinRoot))

```

Comment:
To confirm this means no log file correct?

Reply:
This means to telemetry. We will still get local logs.

Reply:
In that case probably don't need a timestamp per my other comment

Reply:
Does this change the behavior for all telemetry listeners?

Are we ok with that change for other paths?  I'd expect most would break shortly after anyways, since git commands would fail.

Reply:
> Are we ok with that change for other paths? I'd expect most would break shortly after anyways, since git commands would fail.

Yes - the change I am proposing here is to handle this generally (with the degradation that it will not report telemetry). From looking at how several commands construct the tracer, I think today we will generally crash if the tracer fails to find the git installation, and degrading to at least not crash seemed preferable.

If we want to do more around this, (I could see maybe we want to log when the tracer is not able to look up the telemetry switch), I propose we do that in a followup issue. This was our first report of this behavior, so it doesn't seem like a wide spread problem.